gpd/p2-max/default.nix

```diff
@@ -1,11 +1,8 @@
-{lib, ...}: {
+{
   imports = [
     ../../common/pc/laptop
     ../../common/pc/laptop/ssd
     ../../common/cpu/intel
     ../../common/cpu/intel/kaby-lake
   ];
-
-  # HiDPI settings
-  hardware.video.hidpi.enable = lib.mkDefault true;
```

Comment:
We need to keep this option for 22.11 and older. Please add this option back for those.

Reply:
`lib.versionOlder (lib.versions.majorMinor lib.version) "22.11"`